src/assertions.cpp

```diff
@@ -174,6 +175,7 @@ class AssertExtractor {
     }
 };
 
+/// Annotate the CFG by adding in assertions before every memory instruction.
```

Comment:
This is incomplete. We add explicit assertions for all the preconditions of any instruction. For example, jump instructions are asserted not to compare numbers and pointers, or pointers to potentially distinct memory regions.

Reply:
Updated